.gitignore

```diff
@@ -0,0 +1,114 @@
+# container ids
+*.running_container_id*
+
+# Created by https://www.gitignore.io/api/python,pycharm
```

Comment:
Ok I see, then we should not use this generated gitignore as it is not reasonable to add such multiple files for only a gitignore.